index.html

```diff
@@ -246,5 +246,29 @@ <h2 class="abt">
             </div>
         </section>
     </main>
+    <footer>
+        <section class="footer">
+            <div class="footer-bg-img"></div>
+            <div class="contact-details">
+                <h2 class="contact-me">Contact me</h2>
+                <p class="info">
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it 
+                </p>
+            </div>
+            <form class="contact-form" action="https://formspree.io/f/xrgjqpwb">
```

Comment:
- [ ] Kindly make sure to include a post method in your form by adding `method="POST"` attribute, this will enable form data to be sent to formspree.